internal/utils/copy.go

```diff
@@ -0,0 +1,36 @@
+package utils
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"path"
+)
+
+func CopyFile(src, dst string) error {
```

Comment:
Please describe and comment this function.

Reply:
Done 988b5af274e78dc015440cf9beff81710a8a98bf